NavBar.html

```diff
@@ -0,0 +1,142 @@
+        </ol> 
+    </div>
+</div>
+
+<script>
+    import Search from './Search.html'
+
+    export default {
+        components: {
+            Search
+        },
+        /**
+         * Component-wide attributes.
+         * get these with this.get('foo')
+         * and set them with this.set({foo: bar})
+         */
+        data () {
+            return {
+                crumbs: [],
+                currentLevel: 2,
+                // Example value provided here
+                geography: {
+                    state: {
+                        name: 'California'
+                    },
+                    county: {
+                        name: 'San Francisco'
+                    },
+                    zcta: {
+                        id: 94122
+                    }
+                },
+            }
+        },
+        /**
+         * The names of the parameter(s) for these "computed properties" correspond
+         * to attribute(s) in the "data" section (above).
+         * When one of these attributes changes, the property is recomputed.
+         */
+        computed: {
+            currentLevel: (crumbs) => (crumbs.length + 1)
+        },
+        methods: {
+            /**
+             * Pulls a list of breadcrumbs based on the geocode information
+             * contained within the given geography.
+             * @param {Object} geography
+             * @return {Array<string>}
+             */
+            getBreadcrumbsFromGeography = (geography) => {
+                if (!geography) return []
+                const crumbs = []
+                if (geography['state']) {
+                    const crumb = geography['state'].name
+                    crumbs.push(crumb)
+                }
+                if (geography['county']) {
+                    const crumb = geography['county'].name
+                    crumb += ' (county)'
+                    crumbs.push(crumb)
+                }
+                if (geography['city']) {
+                    const crumb = geography['city'].name
+                    crumb += ' (city)'
+                    crumbs.push(crumb)
+                }
+                if (geography['zcta']) {
+                    const crumb = geography['state'].id
+                    crumb += ' (zip code)'
+                    crumbs.push(crumb)
+                }
+                return crumbs
+            },
+            /**
+             * Opens the help modal
+             */
+            openHelpModal () {
+                this.get('helpModal').modal('open')
+            },
+            /**
+             * Navigates to the level corresponding to the clicked breadcrumb
+             */
+            goToLevel (level) {
+                const currentLevel = this.get('currentLevel')
+                if (level === currentLevel) return
+                // TODO: in a different PR, apply the level changes
+                // store.dispatch(...)
```

Comment:
I don't think it's the best practice to implement a partially complete feature as part of the PR. Let's either leave this functionality out entirely, or implement it completely so that it works properly. We can review the issue completion requirements to see whether this feature is in scope for the current PR.